Add categories for Dapr, Conductor and Catalyst (#1)
diagrid-labs/.github: profile/README.md

# 👋 Welcome to Diagrid Labs

> This is the place where you can find sample code and demo apps created by Diagrid.

Reviewer: ```suggestion
Hands-on workshops, sample applications and demos all created by Diagrid.
```

> ## Learn Dapr

Reviewer: Can we add the dapr/conductor/catalyst logos here, on their respective titles?

> [Dapr](https://dapr.io/) provides integrated APIs for communication, state, and workflow. Dapr leverages industry best practices for security, resiliency, and observability, so you can focus on your code.

Reviewer: ```suggestion
[Dapr](https://dapr.io/) is a set of APIs for communication, state, and workflow for building distributed systems. Dapr leverages industry best practices for security, resiliency, and observability, so you can focus on business logic.
```

> - [Learn Dapr](https://github.com/diagrid-labs/learn-dapr) - A collection of resources to learn Dapr.

Reviewer: ```suggestion
- [Learn Dapr](https://github.com/diagrid-labs/learn-dapr) - Start here if you're brand new to Dapr! A curated collection of resources to learn about OSS Dapr.
```

> - [Dapr Workshop](https://github.com/diagrid-labs/dapr-workshop) - A self-paced Dapr workshop available in .NET and Python.

Reviewer: ```suggestion
- [Dapr Workshop](https://github.com/diagrid-labs/dapr-workshop) - A self-paced hands-on Dapr workshop available in .NET and Python. No previous Dapr experience required.
```

> - [Dapr Workflow Demos](https://github.com/diagrid-labs/dapr-workflow-demos) - Demos of the workflow patterns built with Dapr Workflow in .NET.

Reviewer: ```suggestion
- [Dapr Workflow Demos](https://github.com/diagrid-labs/dapr-workflow-demos) - Sample .NET apps showcasing workflow architecture patterns built with the Dapr Workflow API.
```

> - [Dapr Actor Demos](https://github.com/diagrid-labs/dapr-actor-demos) - Demos of the Dapr Actor Model in .NET.

Reviewer: ```suggestion
- [Dapr Actor Demos](https://github.com/diagrid-labs/dapr-actor-demos) - Sample .NET apps using the Dapr Actor API.
```

## Learn Catalyst

> [Diagrid Catalyst](https://www.diagrid.io/catalyst) provides powerful APIs for building microservices and is backed by the Dapr CNCF project.

Reviewer: ```suggestion
[Diagrid Catalyst](https://www.diagrid.io/catalyst) provides a fully managed offering of the Dapr APIs for building microservices on any platform.
```

> - [Catalyst Docs](https://docs.diagrid.io/catalyst/) - Catalyst documentation on the docs.diagrid.io site.

Reviewer: ```suggestion
- [Catalyst Quickstart](https://docs.diagrid.io/catalyst/quickstarts) - Get started immediately with Catalyst on the docs site.
```

> - [Catalyst Order Workflow](https://github.com/diagrid-labs/catalyst-order-workflow) - A demo that shows the capabilities provided by all five Catalyst APIs through a Python-based order processing workflow.

Reviewer: ```suggestion
- [Catalyst Order Workflow](https://github.com/diagrid-labs/catalyst-order-workflow) - A sample Python app showing the capabilities provided by all five Catalyst APIs in an order processing workflow.
```

> - [Catalyst Workflow Saga](https://github.com/diagrid-labs/catalyst-workflow-saga) - A Python demo app that shows you how to implement the AWS reference architecture for Saga orchestration using Catalyst Workflow.

Reviewer: ```suggestion
- [Catalyst Workflow Saga](https://github.com/diagrid-labs/catalyst-workflow-saga) - A sample Python app showing how to implement the AWS reference architecture for Saga orchestration using Catalyst Workflow.
```

> - [Catalyst Reference Apps](https://github.com/diagrid-labs/catalyst-reference-apps) - Deployable reference architectures that leverage the Catalyst APIs from applications and services running in AWS.

Reviewer: ```suggestion
- [Catalyst Reference Apps](https://github.com/diagrid-labs/catalyst-reference-apps) - Sample apps in various languages leveraging the Catalyst APIs for common reference architectures, deployable on AWS.
```

> - [Catalyst Place](https://github.com/diagrid-labs/catalyst-place) - Place is a small Go application powered by Diagrid Catalyst that replicates some functionality from the famous r/place event.

Reviewer: ```suggestion
- [Catalyst Place](https://github.com/diagrid-labs/catalyst-place) - A sample Go app powered by Diagrid Catalyst that replicates some functionality from the famous r/place event.
```

## Learn Conductor

> [Diagrid Conductor](https://www.diagrid.io/conductor) automates the operational management of Dapr on Kubernetes, enhancing observability, improving reliability, and enforcing security best practices. Conductor is a SaaS solution that connects securely to any Kubernetes cluster, automates routine activities like upgrades, and improves uptime of your Dapr applications with monitoring, alerting and 24/7 support. 

Reviewer: ```suggestion
[Diagrid Conductor](https://www.diagrid.io/conductor) automates the operational management of Dapr on Kubernetes, enhancing observability, improving reliability, and enforcing security best practices. Conductor is a SaaS solution that connects securely to any Kubernetes cluster with plans including Dapr OSS Support. 
```

> - [Conductor Docs](https://docs.diagrid.io/conductor/) - Conductor documentation on the docs.diagrid.io site.

Reviewer: ```suggestion
- [Conductor Quickstart](https://docs.diagrid.io/conductor/getting-started/quickstart) - Get started immediately with Conductor on the docs site.
```

> - [Conductor Pizza Store](https://github.com/diagrid-labs/conductor-pizza-store) - This repository contains an example for a distributed Pizza Store application using Kubernetes, Dapr, Spring Boot and Testcontainers.

Reviewer: ```suggestion
- [Conductor Pizza Store](https://github.com/diagrid-labs/conductor-pizza-store) - A sample Java app implementing a distributed Pizza Store application on Kubernetes, using Dapr, Spring Boot and Testcontainers.
```
